mender-convert-package

```diff
@@ -364,7 +364,7 @@ MENDER_PARTITION_ALIGNMENT="${MENDER_PARTITION_ALIGNMENT}"
 MENDER_STORAGE_TOTAL_SIZE_MB="${MENDER_STORAGE_TOTAL_SIZE_MB}"
 MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET="12582912"
 MENDER_ARTIFACT_NAME="${artifact_name}"
-DISTRO_FEATURES="${distro_feature}"
+MENDER_FEATURES="${mender_feature}"
```

Comment:
Also, for symmetry, should the variable be named `mender_features` too?

Reply:
@oleorhagen :+1: 

See the force-push, I decided to use `bootloader_feature` and `mender_features` accordingly. Syntactically, it should make more sense now.